configs/all-configurations.nix

```diff
@@ -3,7 +3,10 @@
   # Refer to <https://github.com/ngi-nix/ngipkgs/issues/40>.
   #liberaforms-container = import ./liberaforms/container.nix;
   flarum = {
-    imports = [./flarum];
+    imports = [
+      ./flarum
+      ./dummy.nix
```

Comment:
It's not evident without context why that is necessary. But supposedly this could be done in the outer layer? `dummy.nix` doesn't seem like a helpful name, and maybe if it's really small that config addition can be declared inline at the point where it's applied to each config."

Reply:
How about now, @fricklerhandwerk ?

Reply:
Makes more sense to me!